When writing about command line switches in a man page, care must be
taken to escape all minus signs to prevent them from being interpreted
as dashes by groff.

This patch fixes a couple such problems in the scrotwm man page.
Index: trunk/scrotwm.1
===================================================================
--- trunk.orig/scrotwm.1	2009-10-03 21:12:00.000000000 +0200
+++ trunk/scrotwm.1	2009-10-04 14:17:14.000000000 +0200
@@ -191,7 +191,7 @@
 .It Cm initscr
 initscreen.sh
 .It Cm menu
-dmenu_run -fn $bar_font -nb $bar_color -nf $bar_font_color -sb $bar_border -sf $bar_color
+dmenu_run \-fn $bar_font \-nb $bar_color \-nf $bar_font_color \-sb $bar_border \-sf $bar_color
 .El
 .Pp
 Custom programs in the configuration file are specified as follows:
@@ -454,7 +454,7 @@
 The default quirks are described below:
 .Pp
 .Bl -tag -width "OpenOffice.org N.M:VCLSalFrame<TAB>XXX" -offset indent -compact
-.It Firefox-bin:firefox-bin
+.It Firefox\-bin:firefox\-bin
 TRANSSZ
 .It Firefox:Dialog
 FLOAT
Index: trunk/scrotwm_es.1
===================================================================
--- trunk.orig/scrotwm_es.1	2009-10-04 14:17:23.000000000 +0200
+++ trunk/scrotwm_es.1	2009-10-04 14:18:31.000000000 +0200
@@ -189,7 +189,7 @@
 .It Cm initscr
 initscreen.sh
 .It Cm menu
-dmenu_run -fn $bar_font -nb $bar_color -nf $bar_font_color -sb $bar_border -sf $bar_color
+dmenu_run \-fn $bar_font \-nb $bar_color \-nf $bar_font_color \-sb $bar_border \-sf $bar_color
 .El
 .Pp
 Los programas en la configuración personal, se especifican aca:
@@ -455,7 +455,7 @@
 Los "quirks" (o forzados) por defecto son:
 .Pp
 .Bl -tag -width "OpenOffice.org N.M:VCLSalFrame<TAB>XXX" -offset indent -compact
-.It Firefox-bin:firefox-bin
+.It Firefox\-bin:firefox\-bin
 TRANSSZ
 .It Firefox:Dialog
 FLOAT
